log4j2/src/main/scala/com/lightbend/lagom/internal/log4j2/Log4j2LoggerConfigurator.scala

```diff
@@ -51,34 +69,8 @@ class Log4j2LoggerConfigurator extends LoggerConfigurator {
         if (env.mode == Mode.Dev) DevLog4j2Config else DefaultLog4j2Config
       ))
 
-    val configUrl = explicitResourceUrl orElse explicitFileUrl orElse resourceUrl
-
-    val properties = LoggerConfigurator.generateProperties(env, configuration, optionalProperties)
-
-    configure(properties, configUrl)
+    explicitResourceUrl orElse explicitFileUrl orElse resourceUrl
```

Comment:
It'd be good to document this behaviour. @rcavalcanti I raised a separate issue ()https://github.com/lagom/lagom/issues/916 so we can finish this one.